pkg/driver/controller.go

```diff
@@ -923,6 +923,21 @@ func (cs *controller) validateVolumeCreateReq(req *csi.CreateVolumeRequest) erro
 			"failed to handle create volume request: missing volume capabilities",
 		)
 	}
+
+	for _, volcap := range volCapabilities {
+		// VolumeCapabilities will contain volume mode
```

Comment:
let us also add check for capability.GetBlock() and capability.GetMount(), either of then should be present, otherwise error out.

Reply:
Updated the code.. PTAL